### PR TITLE
SDL2_image: add new library for emulators

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

File: components/library/SDL2_image/Makefile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+COMPONENT_LICENSE_FILE=	LICENSE.txt
+
+TEST_TARGET= $(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
+
+PATH= $(PATH.gnu)
+
+CONFIGURE_OPTIONS += --sysconfdir=/etc
+CONFIGURE_OPTIONS += --libexecdir=$(CONFIGURE_LIBDIR.$(BITS))
+CONFIGURE_OPTIONS += --disable-png-shared
```
Comment (AndWac) [inline components/library/SDL2_image/Makefile:35]: Are these two first settings really needed? I can't see anything in the manifest related to them.

File: components/library/SDL2_image/SDL2_image.p5m [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+file path=usr/include/SDL2/SDL_image.h
+file path=usr/lib/$(MACH64)/cmake/SDL2_image/sdl2_image-config-version.cmake
+file path=usr/lib/$(MACH64)/cmake/SDL2_image/sdl2_image-config.cmake
+link path=usr/lib/$(MACH64)/libSDL2_image-2.0.so.0 \
+    target=libSDL2_image-2.0.so.0.800.2
+file path=usr/lib/$(MACH64)/libSDL2_image-2.0.so.0.800.2
+file path=usr/lib/$(MACH64)/libSDL2_image.a
+link path=usr/lib/$(MACH64)/libSDL2_image.so target=libSDL2_image-2.0.so.0.800.2
```
Comment (AndWac) [inline components/library/SDL2_image/SDL2_image.p5m:32]: With shared libs available we usually don't deliver static libs. Is there a reason that we want it here?

File: components/library/SDL2_image/Makefile (hunk 1 above)
Comment (Toasterson) [reply components/library/SDL2_image/Makefile:35]: No, Removed.

File: components/library/SDL2_image/SDL2_image.p5m (hunk 2 above)
Comment (Toasterson) [reply components/library/SDL2_image/SDL2_image.p5m:32]: Not really. Dropped

File: components/library/SDL2_image/Makefile [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+COMPONENT_CLASSIFICATION=	System/Multimedia Libraries
+COMPONENT_LICENSE=	zlib
+COMPONENT_LICENSE_FILE=	LICENSE.txt
+
+TEST_TARGET= $(NO_TESTS)
+include $(WS_MAKE_RULES)/common.mk
+
+PATH= $(PATH.gnu)
+
+CONFIGURE_OPTIONS += --disable-png-shared
```
Comment (AndWac) [inline components/library/SDL2_image/Makefile:35]: Maybe -disable-static is supported

**Pull request merged**